src/components/Sum.jsx

```diff
@@ -1 +1,5 @@
-// export function Sum() {}
+export function Sum({ a = 0, b = 0 }) {
+  const sum = +a + +b;
+
+  return `Sum of ${a} and ${b} is ${sum}`;
```

Comment:
```suggestion
  return <p>`Sum of ${a} and ${b} is ${sum}`</p>;
```

better move the tag inside the component, so you can use it without wrapping each time into `<p>`